tests/unit/games-dao.test.js

```diff
@@ -0,0 +1,215 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiExclude = require('chai-exclude');
+const _ = require('lodash');
+const proxyquire = require('proxyquire');
+const sinon = require('sinon');
+const { createConnStub } = require('./test-util');
```

Comment:
A blank line is needed between NPM package require section and local file require section.